### DISCUSSION TITLE
As a verified student can't access github copilot.

### DISCUSSION THREAD

**Discussion by Iabur747** [community/community#136823] [GitHub Education] [answered]
### Hi everyone,

When I try to access GitHub Copilot by clicking "Get direct access on the website GitHub Copilot," it always takes me to the Copilot settings page. Why is this happening? Also, I previously added a card, but I can't remove it. As a student, I need to access GitHub Copilot for free. I am a verified student.

**Comment by Hiba550** [comment]
#### Solution

1. **Redirect to Settings Page:**
   - The redirection to the GitHub Copilot settings page usually occurs if your account settings require review or additional setup. This could also happen if your GitHub account is not properly linked to your student benefits.

2. **Accessing GitHub Copilot for Free:**
   - As a verified student, you should be eligible for free access to GitHub Copilot under the GitHub Student Developer Pack. Here’s what you can do:
     - Ensure that your student verification is up-to-date. You can check this in the [GitHub Student Developer Pack page](https://education.github.com/pack).
     - If your account is verified, visit the [GitHub Copilot page](https://github.com/features/copilot) and look for the option to enable Copilot under the Student Developer Pack.
     - If the issue persists, contact [GitHub Support](https://support.github.com) for assistance with enabling your student benefits for Copilot.

3. **Removing the Payment Card:**
   - GitHub doesn’t currently allow the removal of a payment method if it’s the only one on file. However, you can update the card details with another card.
   - If you believe the payment method was added by mistake or you need it removed due to your student status, reach out to GitHub Support for help in removing it and ensuring that your student benefits are correctly applied.

#### Steps to Contact GitHub Support:
- Visit [GitHub Support](https://support.github.com/contact).
- Choose the appropriate category (e.g., "Billing and payments").
- Provide details about your student verification and the issue you're facing with GitHub Copilot.

By following these steps, you should be able to resolve the issue and gain free access to GitHub Copilot as a verified student.

**Comment by DarkPhoenix-04** [comment, marked as answer]
are you using your college id for accessing github?

**Reply by Iabur747** [reply]
Thank you for your response. My issue has been resolved, though.

**Reply by DarkPhoenix-04** [reply]
then i think you can close this discussion
